test/extended/networking/services.go

```diff
@@ -165,7 +165,7 @@ var _ = Describe("[sig-network] services", func() {
 	})
 
 	InBareMetalIPv4ClusterContext(oc, func() {
-		It("ensures external auto assign cidr is configured correctly on the cluster [Serial]", func() {
+		It("ensures external auto assign cidr is configured correctly on the cluster [apigroup:config.openshift.io] [Serial]", func() {
```

Comment:
this group is not present in microshift, right? FYI this test should be skipped there nevertheless

Reply:
just checked - it is not present